### PR TITLE
[Refactor] Imports and project

### PR DESCRIPTION
**Pull request by Herklos**
- Refactor imports
- Change project name to channel

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2020-10-01
+### Update
+- Project name to 'channel'
```
Comment (GuillaumeDSM) [inline CHANGELOG.md:9]: 💯

File: channel/channels/channel_instances.py [hunk 2]
```diff
@@ -17,15 +16,30 @@
...
+        """
+        Create the instance if not already created
+        Return the class instance
+        :param args: the constructor arguments
+        :param kwargs: the constructor optional arguments
+        :return: the class only instance
+        """
+        if cls not in cls._instances:
+            cls._instances[cls] = cls(*args, **kwargs)
+        return cls._instances[cls]
```
Comment (GuillaumeDSM) [inline channel/channels/channel_instances.py:41]: 👍

File: .drone.yml [hunk 3]
```diff
@@ -42,4 +42,4 @@ steps:
     event:
       - tag
     repo:
-      - Drakkar-Software/OctoBot-Channels
+      - Drakkar-Software/channel
```
Comment (GuillaumeDSM) [inline .drone.yml:45]: 💯

File: channel/channels/channel_instances.py [hunk 4]
```diff
@@ -94,6 +108,6 @@ def del_chan_at_id(chan_name, chan_id) -> None:
     try:
         ChannelInstances.instance().channels[chan_id].pop(chan_name, None)
     except KeyError:
-        get_logger(ChannelInstances.__name__).warning(
+        logging.getLogger(ChannelInstances.__name__).warning(
```
Comment (GuillaumeDSM) [inline channel/channels/channel_instances.py:111]: We will need a way to log into octobot loggers when available

File: requirements.txt [hunk 5]
```diff
@@ -1,5 +1,2 @@
 # Setup requirements
 cython==0.29.21
-
-# Drakkar-Software requirements
-OctoBot-Commons>=1.3.45, <1.4
```
Comment (GuillaumeDSM) [inline requirements.txt:5]: 👍

**Review by GuillaumeDSM — COMMENTED** [review]
very good job !

File: channel/channels/channel_instances.py (hunk 4 above)
Comment (Herklos) [reply channel/channels/channel_instances.py:111]: Yes

Comment (Herklos) [reply channel/channels/channel_instances.py:111]: We can probably add a method `getLogger` in octobot commons and try to replace logging from default logging to commons one

**Review by GuillaumeDSM — APPROVED** [review]

**Pull request merged**